controller/backing_image_controller.go

```diff
@@ -376,11 +376,13 @@ func (bic *BackingImageController) handleBackingImageDataSource(bi *longhorn.Bac
 				return err
 			}
 			for _, node := range nodes {
-				if types.GetCondition(node.Status.Conditions, longhorn.NodeConditionTypeSchedulable).Status != longhorn.ConditionStatusTrue {
+				condition := types.GetCondition(node.Status.Conditions, longhorn.NodeConditionTypeSchedulable)
+				if condition.Status != longhorn.ConditionStatusTrue {
```

Comment:
After removing the index change, I think all these changes are unnecessary. (Please revert this change across all the files)